control-plane/grpc/proto/v1/misc/common.proto

```diff
@@ -29,10 +29,8 @@ enum Protocol {
   None = 0;
   // shared as NVMe-oF TCP
   Nvmf = 1;
-  // shared as iSCSI
-  Iscsi = 2;
```

Comment:
This needs to remain for backward compatibility as 1.0 already shipped with this present.